cse-cc-import-runner-app/src/main/java/com/farao_community/farao/cse/import_runner/app/dichotomy/DichotomyRunner.java

```diff
@@ -77,18 +79,78 @@ private NetworkShifter getNetworkShifter(CseRequest request,
                                              CseData cseData,
                                              Network network) throws IOException {
         return new LinearScaler(
-            getZonalScalable(request.getMergedGlskUrl(), network),
-            getShiftDispatcher(request.getProcessType(), cseData, network),
-            SHIFT_TOLERANCE);
+                getZonalScalable(request.getMergedGlskUrl(), network, request.getProcessType()),
+                getShiftDispatcher(request.getProcessType(), cseData, network),
+                SHIFT_TOLERANCE);
     }
 
-    private ZonalData<Scalable> getZonalScalable(String mergedGlskUrl, Network network) throws IOException {
+    public ZonalData<Scalable> getZonalScalable(String mergedGlskUrl, Network network, ProcessType processType) throws IOException {
         ZonalData<Scalable> zonalScalable = fileImporter.importGlsk(mergedGlskUrl, network);
         Arrays.stream(CseCountry.values()).forEach(country -> checkCseCountryInGlsk(zonalScalable, country));
+        handleLsk(network, zonalScalable, processType);
         return zonalScalable;
     }
 
-    private void checkCseCountryInGlsk(ZonalData<Scalable> zonalScalable, CseCountry country) {
+    private void handleLsk(Network network, ZonalData<Scalable> zonalScalable, ProcessType processType) {
```

Comment:
Maybe rename this method, something like addScalableOnLoads or stackScalableOnLoads